packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1799,6 +1792,26 @@ export class DataViewExpander {
         };
     }
 
+    private async getRelationshipSettingDVO(relationship: RelationshipDTO): Promise<RelationshipSettingDVO> {
+        const settingResult = await this.consumption.settings.getSettings({
+            query: {
+                scope: "Relationship",
+                reference: relationship.id
+            }
+        });
+
+        const defaultSetting = { isPinned: false };
```

Comment:
Out of interest: What does this `isPinned` do?

Reply:
I don't know :D

Reply:
According to @Siolto it is nowhere used in the App. So probably it doesn't make sense for it to be here.

Reply:
true, but this is a refactoring PR and has nothing to do with isPinned.

Reply:
Maybe we can use this for contact favorites :)

Reply:
Ok, let's review this and discuss it somewhere else. 👍